light/client.go

```diff
@@ -966,6 +1038,10 @@ func (c *Client) removeWitnesses(indexes []int) error {
 	// order so as to not affect the indexes themselves
 	sort.Ints(indexes)
 	for i := len(indexes) - 1; i >= 0; i-- {
+		// Add witness to blacklist
+		removedWitness := c.witnesses[indexes[i]]
+		c.blacklist[removedWitness.ID()] = time.Now()
```

Comment:
I wound't recommend adding blacklist inside the remove function, to avoid risk instead, we can add a new function for addWitnessToBlacklist and call that whenever we need

Reply:
@yzang2019 split it out into another function `addWitnessToBlacklist` take a look.